Navigation.css

```diff
@@ -0,0 +1,38 @@
+@import url(components.css);
+.Nav_links_li, .nav_links_a, button{
+    font-weight: 500;
+    font-size: 1rem;
+    color:var(--textgray);
+    text-decoration: none;
+
+}
+.nav{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    
+    width: 55rem;
+   
+
```

Comment:
Give consistent spacing

Reply:
will take care from next time! thank you for the Suggestion